arraycontext/impl/pytato/compile.py

```diff
@@ -106,14 +107,15 @@ def _get_arg_id_to_arg_and_arg_id_to_descr(args: Tuple[Any, ...]
     arg_id_to_arg: Dict[Tuple[Any, ...], Any] = {}
     arg_id_to_descr: Dict[Tuple[Any, ...], AbstractInputDescriptor] = {}
 
-    for iarg, arg in enumerate(args):
+    for kw, arg in (tuple(enumerate(args))
+                    + tuple(kwargs.items())):
```

Comment:
Use [itertools.chain](https://docs.python.org/3/library/itertools.html#itertools.chain) to avoid creating tuples?

Reply:
Thanks! Done in  78ff8ff.